### PR TITLE
Fix captain train konnector

### PR DESCRIPTION
**Pull request by ZeHiro**

### CODE REVIEW

**Comment by nicofrand** [comment]
What does it solve ? Mine does not seem to have any issue…

**Comment by ZeHiro** [comment]
Multi trips bills

File: server/konnectors/captain_train.js [hunk 1]
```diff
@@ -228,7 +228,7 @@ function fetchBills(requiredFields, entries, data, next) {
     let linkedPNR = [data.pnrs.find(pnr => pnr.id === proof.pnr_id)];
     try {
       linkedPNR = data.pnrs.filter(
-        pnr => pnr.proof_ids.indexOf(proof.id) !== -1
+        pnr => typeof pnr.proof_ids !== 'undefined' && pnr.proof_ids.indexOf(proof.id) !== -1
```
Comment (nicofrand) [inline server/konnectors/captain_train.js]: Do you expect an array ? Maybe `instanceof Array` would be easier to read than !== 'undefined' ?

File: server/konnectors/captain_train.js [hunk 2]
```diff
@@ -97,7 +97,7 @@ function login(requiredFields, entries, data, next) {
         logger.error(err);
         return next(err);
```
Comment (nicofrand) [inline server/konnectors/captain_train.js]: Maybe you could return next('request error') here instead ?

File: server/konnectors/captain_train.js (hunk 1 above)
Comment (ZeHiro) [inline server/konnectors/captain_train.js]: Yes.

File: server/konnectors/captain_train.js (hunk 2 above)
Comment (ZeHiro) [inline server/konnectors/captain_train.js]: Of course

**Comment by nicofrand** [comment]
LGTM !

**Comment by ZeHiro** [comment]
@nicofrand I let you merge then

**Pull request merged**